pkg/deployment/inventory.go

```diff
@@ -46,6 +46,12 @@ func GenerateNodeSetInventory(ctx context.Context, helper *helper.Helper,
 		utils.LogErrorForObject(helper, err, "Could not resolve ansible group vars", instance)
 		return "", err
 	}
+
+	// add TLS ansible variable
+	if instance.Spec.TLSEnabled != nil && *instance.Spec.TLSEnabled {
+		nodeSetGroup.Vars["tls_certs_enabled"] = "true"
```

Comment:
This is going to be a variable that multiple roles from edpm-ansible would use right? Could we call this edpm_tls_certs_enabled?

Reply:
yes - we can do that